### ISSUE TITLE
DogStatsD link is broken in the wiki

### ISSUE THREAD

**Issue by rodrigc** [VictoriaMetrics/VictoriaMetrics#3042]
**Describe the bug**
The link to DogStatsD is broken in a few places on the wiki.

Looks like GitHub does not allow filing a PR against a wiki, so here is the fix:

```
commit cbd7ebf07ca2e5b11067b7a1275200578fdafccb
Author: Craig Rodrigues <rodrigc@crodrigues.org>
Date:   Mon Aug 29 12:50:12 2022 -0700

    Fix link for DogStatsD

diff --git a/README.md b/README.md
index 89df4b2..7a737f2 100644
--- a/README.md
+++ b/README.md
@@ -323,7 +323,7 @@ See also [vmagent](https://docs.victoriametrics.com/vmagent.html), which can be
 
 ## How to send data from DataDog agent
 
-VictoriaMetrics accepts data from [DataDog agent](https://docs.datadoghq.com/agent/) or [DogStatsD]() via ["submit metrics" API](https://docs.datadoghq.com/api/latest/metrics/#submit-metrics) at `/datadog/api/v1/series` path.
+VictoriaMetrics accepts data from [DataDog agent](https://docs.datadoghq.com/agent/) or [DogStatsD](https://docs.datadoghq.com/developers/dogstatsd/) via ["submit metrics" API](https://docs.datadoghq.com/api/latest/metrics/#submit-metrics) at `/datadog/api/v1/series` path.
 
 Run DataDog agent with `DD_DD_URL=http://victoriametrics-host:8428/datadog` environment variable in order to write data to VictoriaMetrics at `victoriametrics-host` host. Another option is to set `dd_url` param at [DataDog agent configuration file](https://docs.datadoghq.com/agent/guide/agent-configuration-files/) to `http://victoriametrics-host:8428/datadog`.
 
diff --git a/Single-server-VictoriaMetrics.md b/Single-server-VictoriaMetrics.md
index ce8e113..9642d98 100644
--- a/Single-server-VictoriaMetrics.md
+++ b/Single-server-VictoriaMetrics.md
@@ -327,7 +327,7 @@ See also [vmagent](https://docs.victoriametrics.com/vmagent.html), which can be
 
 ## How to send data from DataDog agent
 
-VictoriaMetrics accepts data from [DataDog agent](https://docs.datadoghq.com/agent/) or [DogStatsD]() via ["submit metrics" API](https://docs.datadoghq.com/api/latest/metrics/#submit-metrics) at `/datadog/api/v1/series` path.
+VictoriaMetrics accepts data from [DataDog agent](https://docs.datadoghq.com/agent/) or [DogStatsD](Single-server-VictoriaMetrics.md) via ["submit metrics" API](https://docs.datadoghq.com/api/latest/metrics/#submit-metrics) at `/datadog/api/v1/series` path.
 
 Run DataDog agent with `DD_DD_URL=http://victoriametrics-host:8428/datadog` environment variable in order to write data to VictoriaMetrics at `victoriametrics-host` host. Another option is to set `dd_url` param at [DataDog agent configuration file](https://docs.datadoghq.com/agent/guide/agent-configuration-files/) to `http://victoriametrics-host:8428/datadog`.
 ```

**Comment by valyala** [comment]
@rodrigc , thanks for the bugreport! The links to DogStatsD should be fixed in the commit 1eec2460baa0e75c3cedaef989c77984fad8beb4 .

All the wiki docs are generated from the [docs](https://github.com/VictoriaMetrics/VictoriaMetrics/tree/master/docs) directory, so next time you can send a pull request for doc fixes in this directory.

**Comment by valyala** [comment]
Closing the issue as fixed then.
